Libraries/Core/Timers/JSTimers.js

```diff
@@ -51,11 +51,7 @@ const IDLE_CALLBACK_FRAME_DEADLINE = 1;
 
 const MAX_TIMER_DURATION_MS = 60 * 1000;
 const IS_ANDROID = Platform.OS === 'android';
-const ANDROID_LONG_TIMER_MESSAGE =
-  'Setting a timer for a long period of time, i.e. multiple minutes, is a ' +
-  'performance and correctness issue on Android as it keeps the timer ' +
-  'module awake, and timers can only be called when the app is in the foreground. ' +
-  'See https://github.com/facebook/react-native/issues/12981 for more info.';
+const ANDROID_LONG_TIMER_MESSAGE = '(ADVICE)';
 
```

Comment:
I honestly couldn't justify the need of it being logged to the console, that's why i deleted it; I think we may have to submit two seperate PRs, since these commits are kind of mutually exclusive, even though I initially thought they are related! Lets wait until they get back to us...